src/schedule/ll_schedule.c

```diff
@@ -150,13 +156,39 @@ static void schedule_ll_task_done(struct ll_schedule_data *sch,
 		atomic_read(&sch->domain->total_num_tasks));
 }
 
+/* perf measurement windows size 2^x */
+#define CHECKS_WINDOW_SIZE	10
```

Comment:
It looks to me as a debug feature,
once per second doesn't look very heavy, anyway i suggest putting it as an option in kconfig

Reply:
@jsarha can we give it the same Kconfig menu as the same feature that we have in the Zephyr LL scheduler i.e. the same Kconfig should enable them.
Can we also make the output trace message the same as Zephyr version too (using same units and strings) so that it's easy for developers to compare performance between xtos and Zephyr when optimising. Thanks.

Reply:
I have commit in my own repo for adding kconfig under Debug menu, but is that what you wanted? Or should it be without menu presence, and maybe in the schedule directory with changes?

Reply:
There is now a Kconfig option, are you are happy with it? Can I resolve this conversation?